src/Components/SmartComponents/Dashbar/Dashbar.js

```diff
@@ -1,114 +1,101 @@
 import React from 'react';
-import { Card, Grid, GridItem, Split, SplitItem, StackItem, Stack, Alert, CardBody, Text } from '@patternfly/react-core';
+import { Card, Grid, GridItem, StackItem, Stack, Alert, CardBody, Text } from '@patternfly/react-core';
 import { Main } from '@redhat-cloud-services/frontend-components/Main';
 import { SecurityIcon } from '@patternfly/react-icons';
-import { CVES_ALLOWED_PARAMS } from '../../../Helpers/constants';
+import { CVES_ALLOWED_PARAMS, impactList, CVES_DEFAULT_FILTERS } from '../../../Helpers/constants';
 import { constructFilterParameters, useUrlParams } from '../../../Helpers/MiscHelper';
-import { useDispatch } from 'react-redux';
+import { useDispatch, useSelector } from 'react-redux';
 import { changeCveListParameters } from '../../../Store/Actions/Actions';
 import { FormattedMessage } from 'react-intl';
 import messages from '../../../Messages';
+import propTypes from 'prop-types';
+import { buildActiveFilters, removeFilters } from '../../../Helpers/TableToolbarHelper';
+
+const DashbarItem = ({ title, count, impact, onLinkClick, hasIcon }) => {
+    return (
+        <Card isFullHeight className="card-box">
+            <CardBody>
+                <Text className="pf-u-font-size-lg pf-u-font-weight-bold">
+                    {title}
+                </Text>
+                <Text className="pf-u-font-size-lg pf-u-font-weight-bold pf-u-mt-xs">
+                    <a onClick={onLinkClick}>
+                        {hasIcon && (<SecurityIcon
+                            color={impactList[impact]?.iconColor}
+                            size="sm"
+                            className="pf-u-mr-sm" />
+                        )}
+                        {count}
+                    </a>
+                </Text>
+            </CardBody>
+        </Card>
+    );
+};
+
+DashbarItem.propTypes = {
+    title: propTypes.node,
+    count: propTypes.number.isRequired,
+    impact: propTypes.oneOf(impactList),
```

Comment:
`impactList` is an object:
```
 impactList = {
    Critical: {
        title: intl.formatMessage(messages.impactListCritical),
        color: 'var(--pf-global--danger-color--100)',
        text: impactTextList.critical,
        iconColor: 'var(--pf-global--palette--red-200)'
    },
    Important: {
        title: intl.formatMessage(messages.impactListImportant),
        color: 'var(--pf-global--palette--orange-400)',
        text: impactTextList.important,
        iconColor: 'var(--pf-global--palette--orange-300)'
    },
    Moderate: {
        title: intl.formatMessage(messages.impactListModerate),
        color: 'var(--pf-global--warning-color--200)',
        text: impactTextList.moderate,
        iconColor: 'var(--pf-global--palette--gold-400)'
    },
    Low: {
        title: intl.formatMessage(messages.impactListLow),
        color: 'var(--pf-global--default-color--300)',
        text: impactTextList.low,
        iconColor: 'var(--pf-global--palette--black-600)'
    }
};
```
and you want to do 
`impact: propTypes.oneOf(["Critical", "Important", "Moderate", "Low", "Unknown"])`

`propTypes.oneOf` does not work on an object, but to convert the object to array of its keys you can use
`impact: propTypes.oneOf(Object.keys(impactList))`

Reply:
That makes sense, thank you!